tasklist/client/e2e/tests/task-details.spec.ts

```diff
@@ -156,6 +158,58 @@ test.describe('task details page', () => {
     await expect(formJSDetailsPage.completeTaskButton).not.toBeVisible();
   });
 
+  test('complete zeebe and job worker tasks', async ({
+    page,
+    taskDetailsPage,
+    taskPanelPage,
+    formJSDetailsPage,
+  }) => {
+    await taskPanelPage.openTask('Zeebe_user_task');
+    await taskDetailsPage.unassignButton.click();
+    await expect(formJSDetailsPage.completeTaskButton).toBeDisabled({
+      timeout: 60000,
+    });
+    await taskDetailsPage.assignToMeButton.click();
+    await expect(formJSDetailsPage.completeTaskButton).toBeEnabled({
+      timeout: 60000,
```

Comment:
We don't have these timeouts in any of the other tests, why is this necessary here?